lib/shoulda/matchers/active_record/define_enum_for_matcher.rb

```diff
@@ -664,6 +700,10 @@ def expected_instance_methods
           end
         end
 
+        def expected_intance_methods
+          options[:instance_methods]
+        end
```

Comment:
There's a small typo here, and I think you can name that using the `?` as a suffix.

```suggestion
        def expected_instance_methods?
          options[:instance_methods]
        end
```

Reply:
Ohh yeah, dang, didnt catch that. Thank you.